### ISSUE TITLE
Could not find module in path

### ISSUE THREAD

**Issue by miukimiu** [codesandbox/codesandbox-client#4294]
# 🐛 bug report

## Description of the problem

Hello again! 👋 

After #4285 was merged the issue that was previously fixed on #4269 is back:

```js
ModuleNotFoundError
Could not find module in path: '@elastic/eui/lib/components/icon/assets/arrow_up.js' 
relative to '/node_modules/@elastic/eui/lib/components/icon/icon.js'
```

### Link to sandbox: https://codesandbox.io/s/43t0g

Thanks!

**Comment by CompuIves** [comment]
Ah wow, I know what the issue is, and have pushed a fix which should be live (again) in 20 minutes. You might need to clear your cache to see it work, there was an issue with our validation of the packager which made us use the commonjs version of the lib. Which doesn't work with the dynamic imports.

**Comment by miukimiu** [comment]
Thanks @CompuIves. It's working! 🎉

**Comment by dandrei** [comment]
@CompuIves I don't want to open a new ticket because this one is still fresh.
This template recommended for forking by the `recharts` repo also crashes with the same message:
https://codesandbox.io/s/recharts-issue-template-fl9qf

```
ModuleNotFoundError
Could not find module in path: 'd3-interpolate' relative to '/node_modules/d3-scale/src/continuous.js'
```

**Comment by miukimiu** [comment]
Hi @dandrei,
If you add `d3-interpolate` as a dependency it fixes your issue. See the example: https://codesandbox.io/s/recharts-issue-template-no1yj

**Comment by lbogdan** [comment]
@miukimiu It should work without, though, as `d3-interpolate` is already a dependency of `recharts`.
